test/serving.bash

```diff
@@ -184,3 +161,38 @@ function upstream_knative_serving_e2e_and_conformance_tests {
       --imagetemplate "$image_template"
   fi
 }
+
+function enable_feature_flags {
+  local failed=0
+
+  for feature in "$@"; do
+    echo "Enabling feature: $feature"
+    configure_cm features "$feature":enabled || failed=1
+  done
+  # Allow settings to be picked up
+  sleep 30
+  return $failed
```

Comment:
Midstream we fail tests if anything returns !=0, do we need this here too?

Reply:
No because we already have it. We define flags such as `set -e` which make the tests fail fast. And we don't want to gather the additional state which is done in "fail_test" as we run serverless must gather ourselves.

And for the same reason, we should probably remove all occurrences of `|| failed=1` from this PR. The `failed` flag is redundant here.